src/test/regress/expected/multi_mx_function_call_delegation_0.out

```diff
@@ -373,6 +380,7 @@ SET client_min_messages TO NOTICE;
 drop table mx_call_dist_table_replica;
 SET client_min_messages TO DEBUG1;
 select colocate_proc_with_table('mx_call_func', 'mx_call_dist_table_1'::regclass, 1);
+DEBUG:  function does not have co-located tables
```

Comment:
where do all these messages come from?

Reply:
They're coming from TryToDelegateFunctionCall() because now we mark more of the functions created in test files as distributed (even when there are no workers in the cluster).

Reply:
Because ProcessCreateSchema uses NodeDDLTaskList() function to generate a DDLJob too.